### PR TITLE
Script to delete workflow runs

### PR DESCRIPTION
**Pull request by vigneshmsft**
It is not possible to delete a workflow directly from the GitHub interface,
this script iterates all the workflow runs and deletes them

### CODE REVIEW

File: scripts/delete-workflow-runs.ps1 [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+$headers = @{
+  Authorization = "Bearer $apiToken"
+  Accept = "application/vnd.github.v3+json"
+}
+
+#Get all workflow runs
+$getWorkflowRunsResponse = Invoke-RestMethod $getWorkflowRuns -Headers $headers;
+
+# Delete all workflow runs, this will delete the workflow when all runs are deleted
+$getWorkflowRunsResponse.workflow_runs.foreach {
```
Comment (saliceti) [inline scripts/delete-workflow-runs.ps1:20]: Please remove the white space as it causes a parse error

File: scripts/delete-workflow-runs.ps1 [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env pwsh
```
Comment (saliceti) [inline scripts/delete-workflow-runs.ps1:1]: Can you add execute permission on the script?

Comment (vigneshmsft) [reply scripts/delete-workflow-runs.ps1:1]: done

File: scripts/delete-workflow-runs.ps1 (hunk 1 above)
Comment (vigneshmsft) [reply scripts/delete-workflow-runs.ps1]: done @saliceti

**Review by saliceti — APPROVED** [review]

**Pull request merged**